aten/src/TH/generic/THBlas.cpp

```diff
@@ -17,12 +17,21 @@ TH_EXTERNC void saxpy_(int *n, float *a, float *x, int *incx, float *y, int *inc
 TH_EXTERNC double ddot_(int *n, double *x, int *incx, double *y, int *incy);
 #ifdef BLAS_USE_CBLAS_DOT
 TH_EXTERNC float cblas_sdot(const int n, const float *x, const int incx, const float *y, const int incy);
-#ifndef THBlas_C_sdot_
-#define THBlas_C_sdot_
+TH_EXTERNC void cblas_cdotu_sub(const int n, const void *x, const int incx, const void *y, const int incy, void *dotu);
+TH_EXTERNC void cblas_zdotu_sub(const int n, const void *x, const int incx, const void *y, const int incy, void *dotu);
+
+#ifndef THBlas_cblas_dot_
```

Comment:
why are these symbols being defined?

Reply:
removed in https://github.com/pytorch/pytorch/pull/43148